make/CompileInterimLangtools.gmk

```diff
@@ -67,6 +67,29 @@ java.compiler.interim_EXTRA_FILES := \
 
 TARGETS += $(BUILDTOOLS_OUTPUTDIR)/gensrc/java.compiler.interim/javax/tools/ToolProvider.java
 
+################################################################################
+# Generate interim versions of the Preview.java and TransPatters.java files for
+# the interim langtools modules, which will allow to compile Classfile API.
+
+$(BUILDTOOLS_OUTPUTDIR)/gensrc/jdk.compiler.interim/com/sun/tools/javac/code/Preview.java: \
+    $(TOPDIR)/src/jdk.compiler/share/classes/com/sun/tools/javac/code/Preview.java
+	$(call LogInfo, Generating Preview.java for jdk.compiler.interim)
+	$(call MakeDir, $(@D))
+	$(GREP) -v 'case PATTERN_SWITCH ->' $< > $@
```

Comment:
I understand that you have based this on the example above. There is a huge difference, though. The sed expression just updates the package name to include the new `interim` part. These new additions seems to be designed to filter out actual code. 

Since no reason for this is given in comments, I'm assuming there is some code that either does not compile when building this for the interim JDK, or that gives the incorrect result if included. But that means that now suddenly the makefiles has intricate knowledge about specific lines of code in the source code! That is not a good entanglement to have.

Reply:
To be more specific: is there some way the code in Preview.java and TransPatterns.java can be modified so this transmogrification is not needed?

Reply:
So this will be removed later on in one of the follow up "chicken" PRs?

Reply:
Affected modules participate in preview and have enabled preview instead of CompileInterimLangtools.gmk patch